do.sh

```diff
@@ -389,8 +400,20 @@ function run_test() {
     # Perform a fast cleanup by doing a minimal redeploy.
     init_ovn_fake_multinode
 
-    tester_ip=$(get_tester_ip ${test_file})
-    if ! ansible-playbook ${ovn_fmn_playbooks}/run-tester.yml -i ${hosts_file} --extra-vars "test_file=${test_file} tester_ip=${tester_ip} phys_deployment=${phys_deployment}" ; then
+    cluster_vars=""
+    for var in enable_ssl clustered_db monitor_all use_ovsdb_etcd \
+               node_net datapath_type n_relays n_workers; do
+        cluster_vars="${cluster_vars} $(get_cluster_var ${test_file} ${var})"
+    done
+    echo "-- Cluster vars: ${cluster_vars}"
+
+    if ! ansible-playbook ${ovn_fmn_playbooks}/bringup-cluster.yml \
+           -i ${hosts_file} --extra-vars "${cluster_vars}" ; then
+        die "-- Failed to bring up fake cluster!"
```

Comment:
Should we try to collect some basic logs at this point?

Reply:
Which ones?
FWIW, the output of the playbook will contain errors if there are some, and it will be dumped to the test-log.

Reply:
Well, collect-logs.yml below should work just fine even if only part of the containers have been deployed.

Reply:
Well, I'm not convinced these logs contain anything interesting.  The cluster is not even connected yet.  Installation failures will mostly be in ansible failure logs.

Also, I'm not sure how to gracefully jump over to the log collection.  Do you have some idea how to do that?

Reply:
You make a good point. I'm not sure how to nicely jump to the log collection part.  Let's forget about it for now and revisit it if we ever start having issues for which ansible failure logs are not enough.